openstack/nova/templates/pod-exec-rbac.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if and .Values.placement.enabled (or .Values.placement.unsplit (and .Values.placementdbUser .Values.placementdbPassword .Values.placementdbName)) }}
```

Comment:
We need this, too, if `unsplit` is `true`.

Reply:
It is. As long as placement is enabled and either unsplit is true, or the credentials are set:
>  if and .Values.placement.enabled (or **.Values.placement.unsplit**

Reply:
I don't seem to be able to read. You're absolutely right m)